crates/bevy_app/src/schedule_runner.rs

```diff
@@ -14,7 +14,12 @@ use wasm_bindgen::{prelude::*, JsCast};
 /// Determines the method used to run an [App]'s `Schedule`
 #[derive(Copy, Clone, Debug)]
 pub enum RunMode {
-    Loop { wait: Option<Duration> },
+    /// Indicates that the [App]'s schedule should run repeatedly.
+    Loop {
+        /// Minimum duration to wait after a schedule has completed before repeating.
```

Comment:
```suggestion
        /// Minimum [`Duration`] to wait after a schedule has completed before repeating.
```

Users looking at this will typically want to figure out how they create a `Duration`, so we should link it for them.